uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java

```diff
@@ -510,6 +513,11 @@ void deleteRepositoryFiles(Path path,
                      options);
     }
 
+    private boolean isDotFile(Path path) {
```

Comment:
This is up to you, but I think I would change this for `isIgnored`. Great job @marcuscarvalho

Reply:
I agree, "isIgnored" is more readable. Thanks for your review @adrielparedes :)